src/test/XTagTest.java

```diff
@@ -0,0 +1,27 @@
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.cryptomorin.xseries.XMaterial;
+import com.cryptomorin.xseries.XTag;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Simple integration tests to ensure the XTag wrapper properly works.
+ */
+class XTagTest {
+
+  @Test
+  void checkNonInheritTag() {
+    final XMaterial target = XMaterial.TUBE_CORAL;
+
+    assertTrue(XTag.CORAL_PLANTS.isTagged(target));
+  }
+
+  @Test
+  void checkInheritTag() {
```

Comment:
You should add all tests to `Tests` class. I personally don't see the point of separating these.